src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -22,31 +22,38 @@
 
 //	Bookmarks of places to mine. Add additional bookmarks separated by comma.
 string[] SetMiningSiteBookmark = new[] {
-	"mining_site_bookmark_name",
+	"Asteroid Belts",
 	};
 
 //	Bookmark of location where ore should be unloaded.
-string UnloadBookmark = "station_or_POS_bookmark_name";
+string UnloadBookmark = "home";//home is a custom save location of station in my system who called home 
```

Comment:
if you want i can show you on pc how to use it, and i can listen all your ask about my update